index.js

```diff
@@ -36,7 +36,7 @@ function getLogger( name, loggerOpts ){
     return loggers[ name ];
   }
   else {
-    var logger = new createLogger( name, loggerOpts );
+    var logger = new CreateLogger( name, loggerOpts );
```

Comment:
The way that github formatted this, it looks like CreateLogger is calling CreateLogger.  :boom: